Source/Immutable/Private/Immutable/ImmutablePassport.cpp

```diff
@@ -73,6 +87,14 @@ FString FImmutablePassportZkEvmRequestAccountsData::ToJsonString() const
 }
 
 
+FString FImmutablePassportConnectPKCEData::ToJsonString() const
```

Comment:
Can use the method Praveen created instead of this now.

Reply:
Changed it